src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -223,6 +223,7 @@ public function has($id)
             if ('service_container' === $id
                 || isset($this->aliases[$id])
                 || isset($this->services[$id])
```

Comment:
Ok to ease merging between branches.
Regarding the perf, does it really worth to make the code more complex for such micro-optimizations?